roles/ansible_tower/tasks/main.yaml

```diff
@@ -30,3 +30,13 @@
 
 - name: Set (httpd_can_network_connect_db) flag on and keep it persistent across reboots
   seboolean: name=httpd_can_network_connect_db state=yes persistent=yes
+
+- name: Setup proot to allow access to /etc/tower/
+  lineinfile:
+    dest: /etc/tower/settings.py
+    backrefs: yes
+    regexp: "{{ item.regex }}"
+    line: '\1\2= {{ item.options }}'
+  with_items: proot_options
```

Comment:
Let's do this so that if they don't pass in anything for proot_options it doesn't error:
```
  with_items: proot_options | default([])
```